app/json/schedule.json

```diff
@@ -0,0 +1,149 @@
+[
+	{
+		"name": "JavaScript Conference",
+		"days": [
+			{
+				"day_id":"01",
+				"day_name":"Friday, 20 October",
+				"timetable": [
+					{
+						"time":"8:55",
+						"article":"Morning Welcome",
+						"speaker":
+							{
+								"name":"",
+								"position":""
+								},
+						"about":""
+					},
+					{
+						"time":"9:00-10:00",
+						"article":"The Fault, Dear Brutus (or: Career Advice From a Cranky Old Man)",
+						"speaker":
+							{
+								"name":"Jeffrey Zeldman",
+								"position":"A Book Apart, An Event Apart"
+								},
+						"about":"We have met the enemy, and he is us. Many of the professional problems we blame on boneheaded bosses and clueless coworkers actually come from ourselves. Identify career woes you bring on yourself, and learn to get out of your own way. Make yourself irreplaceable."
+					},
+					{
+						"time":"10:15-11:15",
+						"article":"Icon Design Process",
+						"speaker":
+							{
+								"name":"Jon Hicks",
+								"position":"author Icons: The Icon Handbook"
+								},
+						"about":"Icons are more than just pretty decorative graphics for sites and applications; they are little miracle workers. They summarize and explain actions, provide direction, offer feedback and even break through language barriers.Icons are more than just pretty decorative graphics for sites and applications; they are little miracle workers. They summarize and explain actions, provide direction, offer feedback and even break through language barriers."
+					},
+					{
+						"time":"11:30-12:30",
+						"article":"Designing for Social Behaor",
+						"speaker":
+							{
+								"name":"Sarah Parmenter",
+								"position":"Founder, You Know Who"
+								},
+						"about":"Love it or loathe it, social media has been burned into everything we do. Once more, it’s fallen on the head of the trusty, multiple-hat-wearing web designer to become a social engineer. In addition to everything else we do, it’s now our job to design strategies and integrate them into our projects. It’s not just about connecting these services to our apps or websites. It’s about knowing how to craft together behaviors, user experience, and traditional design elements in a beautifully harmonious way that achieves business goals. From lessons learned while designing a large social network from scratch, and minus the overly familiar buzzwords and marketing cliches, Sarah will teach you how to truly engage with your users. Discover the real value of social media reciprocity, and learn how to make it work for you. "
+					},
+					{
+						"time":"12:30-2:00",
+						"article":"Lunch",
```

Comment:
lunch could have a different 'type', thus no need of empty fields, also, this way you could use different styles for different types